scripts/spidhub-fargate/environments/beta/storage/config/spid-saml-check/v1/configvolume/spid_users.json

```diff
@@ -0,0 +1,562 @@
+		"ivaCode": "",
+		"companyName": "",
+		"registeredOffice": "",
+		"companyFiscalNumber": "",
+		"address": "Via Listz 21 00144 Roma",
+		"domicileStreetAddress": "Via Listz 21",
+		"domicilePostalCode": "00144",
+		"domicileMunicipality": "Roma",
+		"domicileProvince": "RM",
+		"domicileNation": "IT"
+	},
+	{
+		"username": "agid",
+		"password": "password123",
+		"purposeIDType": "2",
+		"spidLevel": 2,
+
+		"spidCode": "SPID-012",
+		"familyName": "",
+		"name": "",
+		"gender": "",
+		"dateOfBirth": "",
+		"countyOfBirth": "",
+		"placeOfBirth": "",
+		"idCard": "",
+		"expirationDate": "2040-06-05",
+		"fiscalNumber": "",
+		"mobilePhone": "",
+		"email": "protocollo@pec.agid.gov.it",
+		"digitalAddress": "protocollo@pec.agid.gov.it",
+		"ivaCode": "97735020584",
+		"companyName": "Agenzia per l'Italia Digitale",
+		"registeredOffice": "Via Listz 21 00144 Roma",
+		"companyFiscalNumber": "TINIT-97735020584",
+		"address": "",
+		"domicileStreetAddress": "",
+		"domicilePostalCode": "",
+		"domicileMunicipality": "",
+		"domicileProvince": "",
+		"domicileNation": ""
+	},
+	{
+		"username": "Louis",
+		"password": "password123",
+		"purposeIDType": "3",
+		"spidLevel": 3,
+
+		"spidCode": "SPID-013",
+		"familyName": "Armstrong",
+		"name": "Louis",
+		"gender": "M",
+		"dateOfBirth": "1931-08-04",
+		"countyOfBirth": "EE",
+		"placeOfBirth": "Z404",
+		"idCard": "librettoPensione KK121212 inps 2008-07-03 2018-07-03",
+		"expirationDate": "2019-11-19",
+		"fiscalNumber": "TINIT-RMSLSO31M04Z404R",
+		"mobilePhone": "3334567890",
+		"email": "SatchmoTrumpet@gmail.com",
+		"digitalAddress": "SatchmoTrumpet@postacertificatatrumpet.it",
+		"ivaCode": "",
+		"companyName": "",
+		"registeredOffice": "",
+		"companyFiscalNumber": "",
+		"address": "Via Listz 21 00144 Roma",
+		"domicileStreetAddress": "Via Listz 21",
+		"domicilePostalCode": "00144",
+		"domicileMunicipality": "Roma",
+		"domicileProvince": "RM",
+		"domicileNation": "IT"
+	},
+	{
+		"username": "montessori",
+		"password": "password123",
+		"purposeIDType": "4",
+		"spidLevel": 3,
+
+		"spidCode": "SPID-014",
+		"familyName": "Montessori",
+		"name": "Maria",
+		"gender": "F",
+		"dateOfBirth": "2003-08-31",
+		"countyOfBirth": "AN",
+		"placeOfBirth": "C615",
+		"idCard": "cartaIdentita ZZ246810 comuneAncona 2017-09-06 2027-08-31",
+		"expirationDate": "2019-09-09",
+		"fiscalNumber": "TINIT-MNTMRA03M71C615V",
+		"mobilePhone": "3469999999",
+		"email": "farelecoseconmetodo@alice.it",
+		"digitalAddress": "montessori@pec.agid.gov.it",
+		"ivaCode": "12345678987",
+		"companyName": "Scuola magistrale Montessori",
+		"registeredOffice": "Via Listz 21 00144 Roma",
+		"companyFiscalNumber": "TINIT-12345678987",
+		"address": "Via Listz 21 00144 Roma",
+		"domicileStreetAddress": "Via Listz 21",
+		"domicilePostalCode": "00144",
+		"domicileMunicipality": "Roma",
+		"domicileProvince": "RM",
+		"domicileNation": "IT"
+	},
+	{
+		"username": "little",
+		"password": "password123",
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)